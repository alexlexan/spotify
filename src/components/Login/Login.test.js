import React from "react";
import Login from "./Login.tsx";

test("shows out of cheese error message", () => {
  const component = mount(<Login />);
  expect(component).toMatchSnapshot();
});
