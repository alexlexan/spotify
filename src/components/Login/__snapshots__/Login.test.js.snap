// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shows out of cheese error message 1`] = `
<Login>
  <div
    className="login"
  >
    <img
      alt=""
      src="https://getheavy.com/wp-content/uploads/2019/12/spotify2019-830x350.jpg"
    />
    <a
      href="https://accounts.spotify.com/authorize?client_id=cff76213089a4e228206c8bf120d67cb&redirect_uri=http://localhost:3000/&scope=user-read-currently-playing%20user-read-recently-played%20user-read-playback-state%20user-top-read%20user-modify-playback-state&response_type=token&show_dialog=true"
    >
      LOGIN TO SPOTIFY
    </a>
  </div>
</Login>
`;
